### DISCUSSION TITLE
Test.sol Missing

### DISCUSSION THREAD

**Discussion by kingmanas** [Cyfrin/foundry-full-course-cu#1160] [General]
Error (6275): Source "forge-std/Test.sol" not found: File not found. Searched the following locations: "/root/Blockbuddy/foundry-fund-me".
 --> test/FundMeTest.t.sol:5:1:
  |
5 | import {Test} from "../forge-std/Test.sol";
  
  
  I am getting this error even after having test.sol file...?

**Comment by 0xusmanf** [comment]
You don't have to include `../` in the path just write `import {Test} from "forge-std/Test.sol";`

**Comment by UcheEdeoga** [comment]
I am also having the same issue "Source "forge-std/Test.sol" not found: File import callback not supported"

**Reply by kingmanas** [reply]
Delete the forge std file and run "forge install foundry-rs/forge-std" again and then the error might be resolved. Also dont forget to import {Test} from "forge-std/Test.sol"

**Reply by UcheEdeoga** [reply]
I tried doing this, but then it still shows same error 😔 : "Source "forge-std/Test.sol" not found: File import callback not supported" I checked and had to change the file path to "import {Test} from "lib/forge-std/src/Test.sol"; and it worked
